specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json

```diff
@@ -1397,7 +1397,8 @@
           "description": "SQL Server license type.",
           "enum": [
             "PAYG",
-            "AHUB"
+            "AHUB",
+            "DR"
```

Comment:
Adding new enum value is breaking change, prefer to change in new version.

Reply:
I will not classify it as a breaking change as we are not adding a new property as such. 
We are adding a new value to an existing Enum and we are not changing the older values. So this should be a backward compatible change.
We would want to keep change in the current version. We are not planning in any new version releases soon and we cannot hold on to this change due to the project timeline.